integration/src/main/resources/servers/catalog-perf.json

```diff
@@ -18,5 +18,24 @@
   "testRunnerServiceAccountFile": "testrunner-sa.json",
 
   "skipDeployment": true,
-  "skipKubernetes": true
+  "skipKubernetes": true,
+
+  "versionScripts": [
```

Comment:
It looks like adding this will make the perf test run rely on the terra-helmfile files being present. We'll have to update our notes on how to run perf tests locally.

Reply:
If you don't need `terra-helmfile` for local test run, you can remove the config for `ReadFromTerraHelmfileRepo` and keep the Git log config.